packages/@best/config/src/utils/resolve-config.ts

```diff
@@ -78,6 +78,10 @@ export function readConfigAndSetRootDir(configPath: string): UserConfig {
         configObject.rootDir = path.dirname(configPath);
     }
 
+    if (!configObject.orgName) {
+        throw new Error('A best project must have a orgName');
```

Comment:
So this is a breaking change because of this error?

Reply:
@nolanlawson yes  it is breaking change we are introducing organizations as a layer on the top of projects